packages/shared/components/inputs/UnitInput.svelte

```diff
@@ -37,4 +38,24 @@
     }
 </script>
 
-<Dropdown2 bind:isFocused value={unit} {items} {onSelect} contentWidth small />
+{#if items.length > 1}
+    <Dropdown2 bind:isFocused value={unit} {items} {onSelect} contentWidth small />
+{:else}
+    <div class="block relative small">
+        <div
+            class="selection relative flex flex-row space-x-0 pl-1 pb-1.5 items-end w-full whitespace-nowrap bg-white dark:bg-gray-800 pr-0 border-transparent"
+        >
+            <Text fontSize="16" fontWeight="semibold" classes="text-gray-800 dark:text-white" smaller>
+                {unit}
+            </Text>
+        </div>
+    </div>
+{/if}
```

Comment:
Can we add this logic to the dropdown component itself, so if there is only one item it will only show the text